Client/Frontend/Autofill/CreditCard/CreditCardSettingsView/CreditCardEditViewModel.swift

```diff
@@ -0,0 +1,48 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+import SwiftUI
+
+class CreditCardEditViewModel: ObservableObject {
+    @Published var firstName: String = ""
+    @Published var lastName: String = ""
+    @Published var errorState: String = ""
+    @Published var enteredValue: String = ""
+    @Published var nameIsValid = true
+    @Published var numberIsValid = true
+    @Published var expirationIsValid = true
+    @Published var nameOnCard: String = "" {
+        didSet (val) {
+            nameIsValid = nameOnCard.isEmpty
+            print("\(val)")
+        }
+    }
+
+    @Published var expirationDate: String = "" {
+        didSet (val) {
+            numberIsValid = true
+            print("\(val)")
```

Comment:
same as above

Reply:
Since, these are placeholder files I will completely remove it :)